pdr_backend/pdr_dashboard/test/resources.py

```diff
@@ -53,6 +53,21 @@ def _input_action(dash_duo, input_id, table_id, input_value, expected_rows):
     assert len(dash_duo.find_elements(f"{table_id} tbody tr")) == expected_rows
 
 
+def _select_dropdown_option(dash_duo, dropdown_id, option_text):
+    dropdown = dash_duo.find_element(dropdown_id)
+    dropdown.click()
+    options = dash_duo.find_elements(".VirtualizedSelectOption")
+    for option in options:
+        if option.text == option_text:
+            option.click()
+            break
```

Comment:
maybe add an exception if nothing is found, it should definitely break the tests instead of doing nothing.

Reply:
it will break the test on the next step if option is not found